miqcli/provider.py

```diff
@@ -171,6 +189,19 @@ def get_id(self, name):
         self.get_resource(name)
         return getattr(self.query, 'id')
 
+    def get_attribute(self, ent_id, attribute):
+        """Get the attribute for the collection entity.
+
+        :param ent_id: Entity ID of Collection
+        :type ent_id: id
+        :param attribute: Attribute to retrieve
+        :type attribute: string
+        :return: Attribute
+        :rtype:
+        """
+        atts = self.get_entity(ent_id, attribute).__getattr__(attribute)
```

Comment:
There could be an exception if the attribute doesn't exist for the specific entity, it would be good to catch this exception.